.pre-commit-config.yaml

```diff
@@ -25,7 +25,7 @@ repos:
     - id: ruff-format
       name: ruff formatting
       description: Run ruff formatting
-      entry: poetry run ruff format --force-exclude
+      entry: ruff format --force-exclude
```

Comment:
The reason it was nested like this, and why we do this in other repos, is so that the pre-commit checks can still run even if the user hasn't acitvated their poetry shell.

EG if I open a terminal and try to commit a change to a python file I get this error.

```
13:34:46 ❯ git commit -m "foo"
check for merge conflicts................................................Passed
check toml...........................................(no files to check)Skipped
check yaml...........................................(no files to check)Skipped
fix end of files.........................................................Passed
trim trailing whitespace.................................................Passed
ruff linting.............................................................Failed
- hook id: ruff-lint
- exit code: 1

Executable `ruff` not found

ruff formatting..........................................................Failed
- hook id: ruff-format
- exit code: 1

Executable `ruff` not found
```
It's not easy for someone who doesn't already have this context to know that in order to commit changes I need to activate a poetry shell